src/renderer/wg_engine/tvgWgRenderData.cpp

```diff
@@ -321,15 +321,15 @@ void WgRenderDataShape::updateMeshes(WgContext& context, const WgPolyline* polyl
         updateBBox(pmin, pmax);
     }
     // generate strokes geometry
-    if ((polyline->pts.count >= 1) && rstroke) {
+    if ((polyline->pts.count >= 1) && rstroke && (rstroke->width > 0.0f)) {
         static WgGeometryData geometryData; geometryData.clear();
         static WgPolyline trimmed;
         // trim -> split -> stroke
-        if ((rstroke->dashPattern) && ((rstroke->trim.begin != 0.0f) || (rstroke->trim.end != 1.0f))) {
+        if ((rstroke->dashPattern) && ((rstroke->trim.begin != 0.0f) || (rstroke->trim.end != 1.0f)) && (rstroke->trim.begin <= rstroke->trim.end)) {
             polyline->trim(&trimmed, rstroke->trim.begin, rstroke->trim.end);
             geometryData.appendStrokeDashed(&trimmed, rstroke);
         } else // trim -> stroke
-        if ((rstroke->trim.begin != 0.0f) || (rstroke->trim.end != 1.0f)) {
+        if (((rstroke->trim.begin != 0.0f) || (rstroke->trim.end != 1.0f)) && (rstroke->trim.begin <= rstroke->trim.end)) {
```

Comment:
hmm... `begin` could be larger then `end`.  will you cover the scenario addtionally?

Reply:
fixed